code/modules/mob/living/living.dm

```diff
@@ -451,9 +451,11 @@
 
 	if(!resting)
 		set_resting(TRUE, FALSE)
+		spec_rest(src, TRUE)
```

Comment:
Ok, here's your issue. 
You're calling here spec_rest. What this actually means is "mob/living/spec_rest" of which a proc does not exist.

The only spec rest that exists is in datum/species/lizard

so, you need to make spec rest defined at the mob level instead. Or, you can make a temporary object like so
`var/datum/species/lizard/L = new()`
and then call the proc like this
`L.spec_rest(src, true)`

Reply:
havent tested this but im like 80% sure itll work